### PR TITLE
Preview-tui ensure pts device for tput in subshell

### PR DESCRIPTION
**Pull request by luukvbaal**
~~We can pass the tty device of the main shell in an env variable rather than writing to a tmpfile.~~

Removing redirection of sub-shells restores `tput` behavior without other workarounds for `ncurses-6.3`.

### CODE REVIEW

File: plugins/preview-tui [hunk 1]
```diff
@@ -451,6 +447,7 @@ else
         printf "\$KITTY_LISTEN_ON not set!\nPlease read Usage in preview-tui."
         cfg=$(stty -g); stty raw -echo; head -c 1; stty "$cfg"
     else
-        toggle_preview "$1" &
+        TTY="$(tty)"
```
Comment (N-R-K) [inline plugins/preview-tui:450]: On zsh `$TTY` seems to be predefined. Could be worth checking.

Comment (luukvbaal) [reply plugins/preview-tui:450]: Yeah I'm aware, didn't think it mattered though.

Comment (luukvbaal) [reply plugins/preview-tui:450]: It's not predefined on `sh` or `bash`, but even if it was we would be overwriting it to the same value.

**Comment by luukvbaal** [comment]
I noticed that the missing file descriptor for the pts device is not created due to error redirection. I assume because both stdin and stdout are redirected for the sub-shell, the pts device is not "inherited" from the main shell.

When removing error redirection,`tput` works as expected (and image previews are the correct dimensions).

Adding WIP label to see if I can find a more elegant fix.

**Comment by jarun** [comment]
Thank you!

**Pull request merged**